store/sqlstore/channel_store.go

```diff
@@ -645,11 +645,11 @@ func (s SqlChannelStore) Update(channel *model.Channel) (*model.Channel, *model.
 
 	// Additionally propagate the write to the PublicChannels table.
 	if err := s.upsertPublicChannelT(transaction, updatedChannel); err != nil {
-		return nil, model.NewAppError("SqlChannelStore.Update", "store.sql_channel.update.upsert_public_channel.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrap(err, "failed to update public channels")
```

Comment:
Let's add a prefix of "upsertPublicChannelT: " in the error message. Also, it would be "failed to upsert .. "

Reply:
Done